orgs/troms.json

```diff
@@ -5,11 +5,7 @@
 
   "supportEmail": "",
 
-  "fylkeskommune": {
-    "logoSrc": "/troms_fylkeskommune.svg",
-    "logoSrcDark": "/troms_fylkeskommune_dark.svg",
-    "name": "Troms fylkeskommune"
-  },
+  "fylkeskommune": {},
```

Comment:
```suggestion
  "fylkeskommune": null,
```

Reply:
I took a look at the type and it should actually be undefined or defined, not null (and definitely not an empty object 😄 ). I've removed the properties from troms.json now.